boards: stm32: Fix User LD1 pin assignment

levietduc0712 opened the pull request:
boards: nucleo_h755zi_q: Fix green LED pin assignment

The green LED (User LD1) on the NUCLEO-H755ZI-Q board was previously configured
to use pin PA5 in the Device Tree source file (nucleo_h755zi_q.dtsi).

However, according to the board's user manual (UM2408), User LD1 can be connected
to either PA5 or PB0 depending on the solder bridge configuration:

PB0: SB65 OFF, SB54 ON (default configuration)  
PA5: SB65 ON, SB54 OFF  

Since PB0 is the default configuration on most boards out of the box, this patch
updates the green_led node to use pin PB0 (&gpiob 0) instead of PA5.

Reference:  
NUCLEO-H755ZI-Q User Manual: https://www.st.com/resource/en/user_manual/um2408-stm32h7-nucleo144-board-mb1364-stmicroelectronics.pdf

Signed-off-by: Le Viet Duc <levietduc0712@gmail.com>

etienne-lms reviewed (commented):
LGTM.
Could you make the  commit message  header line more explicit on modified board? E.g.:
```patch
-boards: stm32: Fix green LED pin assignment
+boards: st: nucleo_h755zi_q: Fix green LED pin assignment

levietduc0712 commented:
Hi @etienne-lms,
I have fixed.

etienne-lms reviewed (approved):
Looks all good to me. Thanks for this fix.

etienne-lms commented:
The commit author is still `levietduc0712 <...>`. Maybe should you update your **.gitconfig** file.

levietduc0712 committed:
boards: st: nucleo_h755zi_q: Fix green LED pin assignment

Update the green LED pin from PA5 to PB0 in the device tree, based on the
default solder bridge configuration in the NUCLEO-H755ZI-Q user manual.

Signed-off-by: Duc Le <levietduc0712@gmail.com>
